src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -9,68 +9,204 @@ import { localize } from 'vs/nls';
+					</div>
 				</div>
-				<div class="section help">
-					<h2 class="caption">${escape(localize('welcomePage.help', "Help"))}</h2>
-					<ul>
-						<li><a href="https://aka.ms/get-started-azdata">${escape(localize('welcomePage.gettingStarted', "Getting started"))}</a></li>
-						<li><a href="https://aka.ms/azuredatastudio">${escape(localize('welcomePage.productDocumentation', "Documentation"))}</a></li>
-						<li><a href="https://github.com/Microsoft/azuredatastudio/issues/new/choose">${escape(localize('welcomePage.reportIssue', "Report issue or feature request"))}</a></li>
-						<li><a href="https://github.com/Microsoft/azuredatastudio">${escape(localize('welcomePage.gitHubRepository', "GitHub repository"))}</a></li>
-						<li><a href="https://aka.ms/azuredatastudio-releasenotes">${escape(localize('welcomePage.releaseNotes', "Release notes"))}</a></li>
-					</ul>
+				<div class="ads_homepage__section section header hero">
+					<div class="row start">
+						<div class="header__top_nav">
+							<div class="flex">
+								<div class="icon"></div>
+								<div>
+									<h1>Azure Data Studio</h1>
+									<div class="flex btn_container">
+										<div>
+											<button id="dropdown_btn" class="btn btn--primary dropdown" role="navigation" aria-haspopup="true" aria-controls="dropdown">
+												<div class="dropdown__text" style="pointer-events: none;">
+													<span>${escape(localize('welcomePage.new', "New"))}</span><i class="icon--arrow_down"></i>
+												</div>
+											</button>
+											<nav role="navigation" class="dropdown_nav">
+												<ul id="dropdown" class="dropdown-content" aria-hidden="true" aria-label="submenu" role="menu" aria-labelledby="dropdown_btn">
+													<li role="none"><a role="menuitem" tabIndex="-1" class="move" href="command:registeredServers.addConnection">${escape(localize('welcomePage.newConnection', "New connection"))}</a></li>
+													<li role="none"><a role="menuitem" tabIndex="-1" class="move" href="command:workbench.action.files.newUntitledFile">${escape(localize('welcomePage.newQuery', "New query"))}</a></li>
+													<li role="none"><a role="menuitem" tabIndex="-1" class="move" href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
+													<li role="none" id="dropdown_mac-only"><a role="menuitem" tabIndex="-1" class="move mac-only" href="command:workbench.action.files.openLocalFileFolder">${escape(localize('welcomePage.openFileMac', "Open file"))}</a></li>
+													<li role="none" id="dropdown_windows_linux-only"><a role="menuitem" tabIndex="-1" class="move windows-only linux-only" href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
```

Comment:
there are a lot of the mixed use of _ and - and even double underscore and double hyphen, we have no places in ads doing that. do you mind unify them to only use _?

Reply:
fixed